Source/CarthageKit/Git.swift

```diff
@@ -434,28 +434,39 @@ public func submodulesInRepository(repositoryFileURL: NSURL, revision: String =
 		}
 }
 
+/// Determines whether any reference exists for the given pattern in the given
+/// repository.
+///
+/// If the specified file URL does not represent a valid Git repository, `false`
+/// will be sent.
+public func referenceExistsInRepository(repositoryFileURL: NSURL, pattern: String) -> SignalProducer<Bool, NoError> {
```

Comment:
Can we make this `internal`? I'd like to default to that and only make it `public` if necessary or requested.